src/coreclr/jit/codegenxarch.cpp

```diff
@@ -3269,8 +3269,14 @@ void CodeGen::genCodeForInitBlkUnroll(GenTreeBlk* node)
         }
     }
 
+    // Handle the non-SIMD remainder by overlapping with previously processed data if needed
     if (size > 0)
     {
+        assert(size <= REGSIZE_BYTES);
+
+        // Round down to the closest power of two (1, 2 or 4)
```

Comment:
```suggestion
        // Round up to the closest power of two (1, 2, 4 or 8)
```
?

Reply:
Ah, correct, round up 🙂 my logic is not entirely correct - fixing